cmd/ui/src/views/Explore/utils.ts

```diff
@@ -119,31 +117,12 @@ const initGraphNodes = (
         if (glyphImage) {
             nodeParams.type = 'glyphs';
             nodeParams.glyphs.push({
-                location: GlyphLocation.TOP_RIGHT,
+                location: node.isOwnedObject ? GlyphLocation.BOTTOM_RIGHT : GlyphLocation.TOP_RIGHT,
                 image: glyphImage,
                 ...themedOptions.glyph.colors,
             });
         }
 
-        // Tier zero nodes should be marked with a gem glyph
-        if (node.isTierZero) {
-            nodeParams.type = 'glyphs';
-            nodeParams.glyphs.push({
-                location: GlyphLocation.TOP_RIGHT,
-                image: themedOptions.glyph.tierZeroGlyph.url || '',
-                ...themedOptions.glyph.colors,
-            });
-        }
-
-        if (node.isOwnedObject) {
-            nodeParams.type = 'glyphs';
-            nodeParams.glyphs.push({
-                location: GlyphLocation.BOTTOM_RIGHT,
-                image: themedOptions.glyph.ownedObjectGlyph.url || '',
-                ...themedOptions.glyph.colors,
-            });
-        }
-
```

Comment:
as discussed, this should be put back to support both glyphs when applicable and support added to BHE